src/optimizer/rewriter/query_rewrite_term.c

```diff
@@ -1531,31 +1532,44 @@ qo_fold_is_and_not_null (PARSER_CONTEXT * parser, PT_NODE ** wherep)
 	    }
 
 	  db_make_int (&value, truefalse);
-	  fold = pt_dbval_to_value (parser, &value);
-	  if (fold == NULL)
-	    {
-	      return;
-	    }
-
-	  fold->type_enum = node->type_enum;
-	  fold->info.value.location = node->info.expr.location;
-	  pr_clear_value (&value);
-	  /* replace IS NULL/IS NOT NULL node with newly created VALUE node */
-	  if (prev)
+	}
+      else
+	{
+	  if (node->info.expr.op == PT_IS_NOT_NULL && pt_check_not_null_constraint (parser, from, node_prior))
 	    {
-	      prev->next = fold;
+	      db_make_int (&value, 1);
```

Comment:
가독성을 위해 1대신 TRUE를 사용하면 어떨까요?
```suggestion
	      db_make_int (&value, TRUE);
```

Reply:
[81e7698](https://github.com/CUBRID/cubrid/pull/6401/commits/81e7698fc36d661ac835b21cb7d2b17815e55c82) 에서 수정했습니다.